application/controllers/admin/surveyadmin.php

```diff
@@ -1265,6 +1265,10 @@ public function editlocalsettings($iSurveyID)
      */
     public function copy()
     {
+        if (!Permission::model()->hasGlobalPermission('surveys', 'create')) {
```

Comment:
😢 403 error …

Reply:
What do you mean?

Reply:
Send a 403 error : https://bugs.limesurvey.org/view.php?id=16470

Some reason why it's better are in the comments , some issue take really more time without a clean error : https://bugs.limesurvey.org/view.php?id=16470#c60490

Reply:
Done